packages/manager/modules/iam/src/components/resourceSelect/translations/Messages_fr_FR.json

```
@@ -0,0 +1,5 @@
+{
+  "iam_resource_select_resource_types_error_required": "Veuillez sélectionner un ou plusieurs types de produits.",
+  "iam_resource_select_resource_types_error_resources": "Aucune ressource n'est disponible pour ce ou ces types de produits.",
+  "iam_resource_select_resources_error_required": "Veuillez sélectionner une ou plusieures ressources"
```

```suggestion
  "iam_resource_select_resources_error_required": "Veuillez sélectionner une ou plusieurs ressources"
```
I think that "plusieurs" is invariable in French.